examples/routes/emailLinkRoutes.js

```diff
@@ -0,0 +1,53 @@
+const express = require("express");
+
+module.exports = (corbado) => {
+  const router = express.Router();
+  const emailLinkService = corbado.emailLinkService;
```

Comment:
Do the services not need to be instantiated with "new SERVICE()"? (applies for all services)